--- config/ola.m4
+++ config/ola.m4
@@ -24,6 +24,11 @@
 AC_REQUIRE_CPP()
 PKG_CHECK_MODULES(libprotobuf, [protobuf >= $1])
 
+AC_MSG_CHECKING([protobuf library version])
+PROTOBUF_VERSION=`pkg-config --modversion protobuf`;
+AC_MSG_RESULT([$PROTOBUF_VERSION])
+AC_SUBST([PROTOBUF_VERSION])
+
 AC_SUBST([libprotobuf_CFLAGS])
 
 AC_ARG_WITH([protoc],
@@ -44,28 +49,7 @@
   AC_MSG_ERROR([cannot find 'protoc' program]);
 elif test -n "$1" ; then
   AC_MSG_CHECKING([protoc version])
-  [protoc_version=`$PROTOC --version 2>&1 | grep 'libprotoc' | sed 's/.*\([0-9][0-9]*\.[0-9][0-9]*\.[0-9][0-9]*\).*/\1/g'`]
-  [required=$1]
-  [required_major=`echo $required | sed 's/[^0-9].*//'`]
-  [required_minor=`echo $required | sed 's/[0-9][0-9]*\.\([0-9][0-9]*\)\.[0-9][0-9]*/\1/'`]
-  [required_patch=`echo $required | sed 's/^.*[^0-9]//'`]
-  [actual_major=`echo $protoc_version | sed 's/[^0-9].*//'`]
-  [actual_minor=`echo $protoc_version | sed 's/[0-9][0-9]*\.\([0-9][0-9]*\)\.[0-9][0-9]*/\1/'`]
-  [actual_patch=`echo $protoc_version | sed 's/^.*[^0-9]//'`]
-
-  protoc_version_proper=`expr \
-    $actual_major \> $required_major \| \
-    $actual_major \= $required_major \& \
-    $actual_minor \> $required_minor \| \
-    $actual_major \= $required_major \& \
-    $actual_minor \= $required_minor \& \
-    $actual_patch \>= $required_patch `
-
-  if test "$protoc_version_proper" = "1" ; then
-    AC_MSG_RESULT([$protoc_version])
-  else
-    AC_MSG_ERROR([protoc version too old $protoc_version < $required]);
-  fi
+  AC_MSG_RESULT([Version check skipped])
 fi
 
 AC_ARG_WITH([ola-protoc-plugin],
@@ -93,7 +77,7 @@
   LIBS=$SAVED_LIBS
 fi
 AC_SUBST([OLA_PROTOC])
-AM_CONDITIONAL(BUILD_OLA_PROTOC_PLUGIN, test "${with_ola_protoc_plugin}" == "no")
+AM_CONDITIONAL(BUILD_OLA_PROTOC_PLUGIN, test "${with_ola_protoc_plugin}" = "no")
 ])
 
 
@@ -144,7 +128,7 @@
   fi
 
   # If dependencies are not met...
-  if test "$3" == "no"; then
+  if test "$3" = "no"; then
     # ...and the user has explicitly requested this plugin to be enabled,
     # error out.
     if test "${enable_plugin}" = "yes"; then
