--- a/protoc/Makefile.mk
+++ b/protoc/Makefile.mk
@@ -1,22 +1,8 @@
 # Programs
 #########################
 if BUILD_OLA_PROTOC_PLUGIN
-noinst_PROGRAMS += protoc/ola_protoc_plugin
-
-protoc_ola_protoc_plugin_SOURCES = \
-    protoc/CppFileGenerator.cpp \
-    protoc/CppFileGenerator.h \
-    protoc/CppGenerator.cpp \
-    protoc/CppGenerator.h \
-    protoc/GeneratorHelpers.cpp \
-    protoc/GeneratorHelpers.h \
-    protoc/ServiceGenerator.cpp \
-    protoc/ServiceGenerator.h \
-    protoc/StrUtil.cpp \
-    protoc/StrUtil.h \
-	protoc/ola-protoc-generator-plugin.cpp
-protoc_ola_protoc_plugin_CXXFLAGS = $(COMMON_PROTOBUF_CXXFLAGS)
-protoc_ola_protoc_plugin_LDADD = $(libprotobuf_LIBS) -lprotoc
+# Plugin disabled
+noinst_PROGRAMS =
 
 else
 
