--- a/config/ola.m4
+++ b/config/ola.m4
@@ -50,32 +50,4 @@
elif test -n "$1" ; then
  AC_MSG_CHECKING([protoc version])
+  AC_MSG_RESULT([skipped version check])
-  # Since v20.x we only get effectively the minor and patch versions out of protoc.
-  # Treat them as major and minor and everything should keep working indefinitely.
-  # See https://protobuf.dev/support/version-support/
-  # So we've got either of these:
-  # libprotoc 2.4.1
-  # libprotoc 23.3
-  # The first sed ensures all versions have major, minor, patch, by adding a .0 on the end of ones missing it
-  [protoc_version=`$PROTOC --version 2>&1 | grep 'libprotoc' | sed 's/\([^\.0-9][0-9][0-9]*\.[0-9][0-9]*\)$/\1\.0/g' | sed 's/[^0-9]*\([0-9][0-9]*\.[0-9][0-9]*\.[0-9][0-9]*\).*/\1/g'`]
-  [required=$1]
-  [required_major=`echo $required | sed 's/[^0-9].*//'`]
-  [required_minor=`echo $required | sed 's/[0-9][0-9]*\.\([0-9][0-9]*\)\.[0-9][0-9]*/\1/'`]
-  [required_patch=`echo $required | sed 's/^.*[^0-9]//'`]
-  [actual_major=`echo $protoc_version | sed 's/[^0-9].*//'`]
-  [actual_minor=`echo $protoc_version | sed 's/[0-9][0-9]*\.\([0-9][0-9]*\)\.[0-9][0-9]*/\1/'`]
-  [actual_patch=`echo $protoc_version | sed 's/^.*[^0-9]//'`]
-
-  protoc_version_proper=`expr \
-    $actual_major \> $required_major \| \
-    $actual_major \= $required_major \& \
-    $actual_minor \> $required_minor \| \
-    $actual_major \= $required_major \& \
-    $actual_minor \= $required_minor \& \
-    $actual_patch \>= $required_patch `
-
-  if test "$protoc_version_proper" = "1" ; then
-    AC_MSG_RESULT([$protoc_version])
-  else
-    AC_MSG_ERROR([protoc version too old $protoc_version < $required]);
-  fi
fi
